snc.sh

```diff
@@ -197,5 +198,16 @@ retry ${OC} patch config.imageregistry.operator.openshift.io/cluster --patch '{"
 # Delete the pods which are there in Complete state
 retry ${OC} delete pod --field-selector=status.phase==Succeeded --all-namespaces
 
+# Add a user developer:developer with htpasswd identity provider and give it sudoer role
+retry ${OC} create secret generic htpass-secret --from-literal=htpasswd=${DEVELOPER_USER_PASS} -n openshift-config
+retry ${OC} apply -f oauth_cr.yaml
+retry ${OC} create clusterrolebinding developer --clusterrole=sudoer --user=developer
+
+# Replace pull secret with a null json string '{}'
+retry ${OC} replace -f pull-secret.yaml
+
+# Remove the Cluster ID with a empty string.
+retry ${OC} patch clusterversion version -p '{"spec":{"clusterID":""}}' --type merge
+
 # Wait for the cluster again to become stable because of all the patches/changes
 wait_till_cluster_stable
```

Comment:
I would move `${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl stop kubelet` right after `wait_till_cluster_stable`, this guarantees there won't be more cluster changes after this point.

Reply:
@cfergeau I am not moving the kubelet stop part in here because what if someone just run snc and try out the cluster?

Reply:
can go at the very beginning of `createdisk.sh` then, with a big comment saying cluster changes must not be done as part of createdisk? (maybe it's already this way, did not check). Can be done in a followup though.